clients/web/src/views/layout/HeaderUserView.js

```diff
@@ -26,11 +26,12 @@ var LayoutHeaderUserView = View.extend({
         'click a.g-logout': logout,
 
         'click a.g-my-folders': function () {
-            router.navigate(`user/${getCurrentUser().id}`, {trigger: true});
```

Comment:
Why is this being changed?

Reply:
It shouldn't be. I will revert this change and the one a few lines below.